test/pageobjects/BookStore.js

```diff
@@ -0,0 +1,42 @@
+class BookStore {
+  get bookStoreBtn() {
+    return $("#item-2*=Book Store");
+  }
+  get profileBtn() {
+    return $("#item-3*=Profile");
+  }
+  get bookGitGuide() {
+    return $("=Git Pocket Guide");
```

Comment:
Please, try to use css selectors, attribute selectors, instead searching by text, cause text might change more often that the selector of an element.